packages/components/psammead-image/src/testHelpers/stories.jsx

```diff
@@ -1,14 +1,16 @@
 import React from 'react';
 import { storiesOf } from '@storybook/react';
+import { boolean } from '@storybook/addon-knobs';
 import notes from '../../README.md';
 import { custom, landscape, portrait, square } from './fixtureData';
 
-export function getProps(image, includeHeight) {
+export function getProps(image, includeHeight, type) {
   const props = {
     alt: image.alt,
     src: image.src,
     srcset: image.srcset,
     width: image.width,
+    fade: type === 'Img' ? boolean('load with fade?', false) : null,
```

Comment:
@rhell we're reusing this story for both amp and image, I set the type for image to `"Img"`, hence the conditional here, because we don't want to show this knob for amp too

Reply:
Do we have confirmation that we don't want to fade images in on amp?

Reply:
yeah, just confirmed from @sareh that we're just doing the fade to just images on canonical pages